solutions/standard/variables.tf

```diff
@@ -151,7 +151,7 @@ variable "powervs_backup_network" {
 variable "powervs_image_names" {
   description = "List of Images to be imported into cloud account from catalog images. Supported values can be found [here](https://github.com/terraform-ibm-modules/terraform-ibm-powervs-workspace/blob/main/docs/catalog_images_list.md)"
```

Comment:
Mention here 'For custom os image import configure the optional parameter 'powervs_custom_images'